life_expectancy/main.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import pandas as pd
+from .load_save import load_data, save_data, LoadJson, LoadTsv
+from .cleaning import clean_data, CleanJson, CleanTsv
+from .my_constants import Country, InputFileType, EnumAction, country_name_from_code
+
+def main(region: Country, file_type: InputFileType)-> pd.DataFrame:
+    """Main function.
+    
+    Args:
+        region (Country): The region to select.
+        file_type (InputFileType): Input file type.
+    """
+
+    file_type_strategy = {'tsv':{'load_file': LoadTsv(), 'clean_data': CleanTsv()},
+                          'json':{'load_file': LoadJson(), 'clean_data': CleanJson()}}
+    print(f'Loading data from {file_type.name} file. '\
```

Comment:
Nice to have:
- Instead of using print statements, consider implementing a logging system that can be configured for different levels of verbosity.

Reply:
Thank you for the suggestion. Which tools would you use to do this?

Reply:
I would just use the logging from python itself, here is the documentation => https://docs.python.org/3/library/logging.html